server/src/lsp-server.ts

```diff
@@ -38,6 +38,7 @@ import { provideOrganizeImports } from './organize-imports';
 import { TypeScriptInitializeParams, TypeScriptInitializationOptions, TypeScriptInitializeResult } from './ts-protocol';
 import { collectDocumentSymbols, collectSymbolInformations } from './document-symbol';
 import { computeCallers, computeCallees } from './calls';
+import { CodeActionKind } from "vscode-languageserver";
```

Comment:
This code base seems to be using single quotes.

(Unfortunately the used linter doesn't check those...)